Documentation/Books/Manual/DataModeling/OperationalFactors.md

```diff
@@ -244,8 +245,7 @@ negatively impact the write performance:
 
 #### Improving Update Query Perfromance
```

Comment:
If the headline on line 235 is now level 2, then this here should be only level 3, shouldn't it? (and possibly subsequent headlines too)